pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/QueryOp.java

```diff
@@ -55,7 +73,87 @@ public void setExpectedResultsFileName(String expectedResultsFileName) {
 
   @Override
   boolean runOp() {
-    System.out.println("Verifying queries in " + _queryFileName + " against results in " + _expectedResultsFileName);
-    return true;
+    System.out.printf("Verifying queries in %s against results in %s\n", _queryFileName, _expectedResultsFileName);
+    try {
+      return verifyQueries();
+    } catch (Exception e) {
+      LOGGER.error("FAILED to verify queries in {}: {}", _queryFileName, e);
+      return false;
+    }
+  }
+
+  boolean verifyQueries()
+      throws Exception {
+    boolean testPassed = false;
+
+    try (BufferedReader queryReader = new BufferedReader(
+        new InputStreamReader(new FileInputStream(_queryFileName), StandardCharsets.UTF_8));
+        BufferedReader expectedResultReader = new BufferedReader(
+            new InputStreamReader(new FileInputStream(_expectedResultsFileName), StandardCharsets.UTF_8))) {
+
+      int passed = 0;
+      int total = 0;
+      int queryLineNum = 0;
+      String query;
+
+      while ((query = queryReader.readLine()) != null) {
+        queryLineNum++;
+        if (shouldIgnore(query)) {
+          continue;
+        }
+
+        JsonNode expectedJson = null;
+        try {
+          String expectedResultLine = expectedResultReader.readLine();
+          while (shouldIgnore(expectedResultLine)) {
+            expectedResultLine = expectedResultReader.readLine();
+          }
+          expectedJson = JsonUtils.stringToJsonNode(expectedResultLine);
+        } catch (Exception e) {
+          LOGGER.error("Comparison FAILED: Line: {} Exception caught while getting expected response for query: '{}'",
+              queryLineNum, query, e);
+        }
+
+        JsonNode actualJson = null;
+        if (expectedJson != null) {
+          try {
+            actualJson = QueryProcessor.postSqlQuery(query);
+          } catch (Exception e) {
+            LOGGER.error("Comparison FAILED: Line: {} Exception caught while running query: '{}'", queryLineNum, query,
+                e);
+          }
+        }
+
+        if (expectedJson != null && actualJson != null) {
+          try {
+            boolean comparisonResult = SqlResultComparator.areEqual(actualJson, expectedJson, query);
+            if (comparisonResult) {
+              passed++;
+              LOGGER.info("Comparison PASSED: Line: {} actual Time: {} ms expected Time: {} ms Docs Scanned: {}",
```

Comment:
We should not be logging expected time, etc. We should not even have it in metadata. We can add some outliers for time later. For now, remove it from the results file and also from this log.
Also, we don't need an INFO log for every passing query. Just add a single  debug log with query, expected result and actual result

Reply:
done